### ISSUE TITLE
bug: rebase instructions don't include aurora

### ISSUE THREAD

**Issue by RealVishy** [ublue-os/aurora#64]
### Describe the bug

The [changelogs.py](https://github.com/ublue-os/aurora/blob/main/.github/changelogs.py) say $IMAGE_NAME

![Image](https://github.com/user-attachments/assets/7a472b23-6ef7-441c-9d1e-a81a70d275e4)

### What did you expect to happen?

i expect it to just say aurora

### Output of `bootc status`

```shell

```

### Output of `groups`

```shell

```

**Comment by ledif** [comment]
I believe changing `$IMAGE_NAME` to just `aurora` here wouldn't be completely correct. The image's name can be one of many variants depending on the currently booted system: `aurora`, `aurora-dx`, `aurora-nvidia`, `aurora-dx-nvidia`, etc.

I don't think we want to give users instructions to rebase to a different image name in these notes, just a different tag.

**Comment by RealVishy** [comment]
AFAIK the changelogs are only for stable, so the tag would be the same. Maybe we can add a sentence above the rebase instructions saying what the image types are ( -dx, -nvidia) etc.

**Comment by RealVishy** [comment]
yeah nvm it's not a bug i just didn't read it properly, the first line figures out your image name and then rebases you to the stable channel for your system. I'll close this.
